python/nano/tutorial/inference/tensorflow/mnist.py

```diff
@@ -0,0 +1,117 @@
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+import numpy as np
+import tensorflow as tf
+from tensorflow import keras
+from keras import layers
+from keras import losses, metrics
+
+from bigdl.nano.tf.keras import Model
+
+# Model / data parameters
+num_classes = 10
+input_shape = (28, 28, 1)
+
+# Load the data and split it between train and test sets
+(x_train, y_train), (x_test, y_test) = keras.datasets.mnist.load_data()
+
+# Scale images to the [0, 1] range
+x_train = x_train.astype("float32") / 255
+x_test = x_test.astype("float32") / 255
+# Make sure images have shape (28, 28, 1)
+x_train = np.expand_dims(x_train, -1)
+x_test = np.expand_dims(x_test, -1)
+print("x_train shape:", x_train.shape)
+print(x_train.shape[0], "train samples")
+print(x_test.shape[0], "test samples")
+
+
+# convert class vectors to binary class matrices
+y_train = keras.utils.to_categorical(y_train, num_classes)
+y_test = keras.utils.to_categorical(y_test, num_classes)
+
+model = keras.Sequential(
+    [
+        keras.Input(shape=input_shape),
+        layers.Conv2D(32, kernel_size=(3, 3), activation="relu"),
+        layers.MaxPooling2D(pool_size=(2, 2)),
+        layers.Conv2D(64, kernel_size=(3, 3), activation="relu"),
+        layers.MaxPooling2D(pool_size=(2, 2)),
+        layers.Flatten(),
+        layers.Dropout(0.5),
+        layers.Dense(num_classes, activation="softmax"),
+    ]
+)
+
+model = Model(inputs=model.inputs, outputs=model.outputs)
+
+model.summary()
+
+batch_size = 128
+epochs = 15
+
+model.compile(loss="categorical_crossentropy",
+              optimizer="adam", metrics=["accuracy"])
+
+model.fit(x_train, y_train, batch_size=batch_size,
+          epochs=epochs, validation_split=0.1)
+
+score = model.evaluate(x_test, y_test, verbose=0)
+print("Test loss:", score[0])
+print("Test accuracy:", score[1])
+
+
+# Quantization using Intel Neural Compressor
+# pip install 'neural-compressor'
+
+# calib_dataset only accept tf.data.Dataset object
+tune_dataset = tf.data.Dataset.from_tensor_slices((x_train, y_train))
+
+# Execute quantization
+q_model = model.quantize(calib_dataset=tune_dataset)
+
+y_test_hat = q_model(x_test)
+loss = float(tf.reduce_mean(
+             losses.categorical_crossentropy(y_test, y_test_hat)))
+categorical_accuracy = metrics.CategoricalAccuracy()
+categorical_accuracy.update_state(y_test, y_test_hat)
+accuracy = categorical_accuracy.result().numpy()
+print("Quantization test loss:", loss)
+print("Quantization test accuracy:", accuracy)
+# Raw model test loss: 0.024767747148871422
+# Raw model test accuracy: 0.9918000102043152
+# Quantized model test loss: 0.02494174614548683
+# Quantized model test accuracy: 0.9917
+# Accuracy loss: 0.01%
```

Comment:
that's reasonable accuracy drop. But we may just use some qualitative way to describe this in comment (e.g. accuracy drop is with on 0.1%). That's because the quantization may bring different result every time you run it.